src/components/Modal/ModalHeader/ModalHeader.tsx

```diff
@@ -10,9 +10,9 @@ import styles from "./ModalHeader.module.scss";
 
 export interface ModalHeaderProps extends VibeComponentProps {
   /**
-   * Title of the modal
+   * Title of the modal - using string is a recommended standard
    */
-  title: string;
+  title: ElementContent;
```

Comment:
let's document the `EditableHeading` example

Reply:
Not sure we want to encourage people to use it more by promoting in the story. I think if there is a need, people will figure it out, if no, they won't even notice - we also used to have this approach in such cases before

Reply:
but if it's a good use case - why not show it? then it also help us to see (and test) the feature